Monika After Story/game/event-handler.rpy

```diff
@@ -2504,9 +2504,9 @@ label call_next_event:
         ):
             #Create a new notif
             if renpy.windows:
-                $ display_notif(m_name, mas_win_notif_quips, "Topic Alerts")
+                $ mas_display_notif(m_name, mas_win_notif_quips, "Topic Alerts")
```

Comment:
can you alias `display_notif` instead of just deleting it? Especially since reacts has been out for a while now.